aws/ecs/packer/to-upload/run.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+set -eu
+
+LOGGER_TAG=""
+
+is_container_running() {
+    CONTAINER_NAME=$1
+    [ "$(docker inspect -f '{{.State.Running}}' $1 2> /dev/null)" = true ]
+}
+
+succeed_or_die() {
+    if ! OUT=$($@ 2>&1); then
+	echo "error: $1 failed: $OUT" | logger -p local0.error -t $LOGGER_TAG
+	exit 1
+    fi
+}
+
+run_scope() {
+    while true; do
+	# verify that scope is not running
+	while is_container_running weavescope; do sleep 2; done
+	# launch scope
+	ARGS=""
+	if [ -e /etc/weave/scope.config ]; then
+	    . /etc/weave/scope.config
+	fi
+	if [ -n "${SERVICE_TOKEN+x}" ]; then
+	    ARGS="$ARGS --service-token=$SERVICE_TOKEN"
+	fi
+	PEERS=$(succeed_or_die /etc/weave/peers.sh)
+	args="$ARGS 127.0.0.1 $PEERS"
```

Comment:
No, we exclude it explicitly since we don't want to use it in the case of Weave.